nameable.rb

```diff
@@ -0,0 +1,6 @@
+# Implement a method called correct_name that will raise a NotImplementedError.
+class Nameable
+  def correct_name
+    raise NotImplementedError
```

Comment:
- [ ]  Great job to raise an error but for a good practice, I would recommend you to attach a message that explain what a user did wrong.  This will make your app more usable friendly.